src/GraphQL/Language/AST/CommentNode.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace GraphQL.Language.AST
+{
+    public class CommentNode : AbstractNode
+    {
+        public CommentNode(string comment)
+        {
+            Comment = comment;
+        }
+
+        public string Comment { get; }
+
+        public override bool IsEqualTo(INode node)
+        {
+            throw new NotImplementedException();
```

Comment:
Why? Every node implements this except `NameNode`.

Reply:
Done